src/dbnode/client/client.go

```diff
@@ -20,33 +20,36 @@
 
 package client
 
-import "sync"
+import (
```

Comment:
Any thoughts on the existing client wrapping multiple sessions (the approach you implemented) vs. having a multi-client that wraps multiple single clients? I haven't put too much thought into it so not sure whats better, just wondering if you considered both

Reply:
Yeah that actually seems better, having given it some thought I can't think of any objections. It would definitely simplify things to not have to pass down the multi options from config -> client -> session.

Reply:
Yeah I guess the `ReplicatedClient` would just return a `ReplicatedSession` anyways